src/Features/Core/Portable/GenerateEqualsAndGetHashCodeFromMembers/FormatLargeBinaryExpressionRule.cs

```diff
@@ -8,66 +8,63 @@
 
 namespace Microsoft.CodeAnalysis.GenerateEqualsAndGetHashCodeFromMembers
 {
-    internal partial class AbstractGenerateEqualsAndGetHashCodeFromMembersCodeRefactoringProvider
+    internal partial class AbstractGenerateEqualsAndGetHashCodeService
     {
-        private partial class GenerateEqualsAndGetHashCodeAction : CodeAction
+        /// <summary>
+        /// Specialized formatter for the "return a == obj.a &amp;&amp; b == obj.b &amp;&amp; c == obj.c &amp;&amp; ...
+        /// code that we spit out.
+        /// </summary>
+        private class FormatLargeBinaryExpressionRule : AbstractFormattingRule
```

Comment:
this diff will look fine if you use https://github.com/dotnet/roslyn/pull/28066/files?w=1